aten/src/ATen/core/type.cpp

```diff
@@ -870,14 +870,29 @@ VaryingShape<Stride> TensorType::computeStrideProps(
   std::vector<size_t> stride_indices(sizes.size());
   std::iota(stride_indices.begin(), stride_indices.end(), 0);
 
+  // Sorting strides in ascending order
+  // Warning: With more than 2 strides of the same size
```

Comment:
nitpick: `2 strides of the same size` is confusing. IIUC, we just can't handle more than 1 dimension with size == 1, this doesn't seem to be saying that.

Reply:
It made perfect sense to me in my mind when writing it for myself. :-D